invenio_rdm_records/resources/config.py

```diff
@@ -80,6 +81,7 @@ def csl_url_args_retriever():
         StringCitationSerializer(url_args_retriever=csl_url_args_retriever),
         headers={"content-type": "text/plain"},
     ),
+    "application/x-bibtex": ResponseHandler(BibtexSerializer()),
```

Comment:
Discussed with him IRL: we don't conflict because he's changing the serializer, which right now is only exposed via the record landing page's export formats. This change just adds one more place to expose the format (via content negotiation on the REST API).